docker/start-temporal.sh

```diff
@@ -3,13 +3,14 @@
 set -eu -o pipefail
 
 : "${SERVICES:=}"
-: "${SERVICE_FLAGS:=}"
 
-if [[ -z ${SERVICE_FLAGS}  && -n ${SERVICES} ]]; then
-    # Convert semicolon (or comma, for backward compatibility) separated string (i.e. "history:matching")
+flags=()
+if [[ -n ${SERVICES} ]]; then
+    # Convert colon (or comma, for backward compatibility) separated string (i.e. "history:matching")
     # to valid flag list (i.e. "--service=history --service=matching").
-    IFS=':,' read -ra SERVICE_FLAGS <<< "${SERVICES}"
-    for i in "${!SERVICE_FLAGS[@]}"; do SERVICE_FLAGS[$i]="--service=${SERVICE_FLAGS[$i]}"; done
+    SERVICES="${SERVICES//:/,}"
+    SERVICES="${SERVICES//,/ }"
```

Comment:
Isn't it time to remove `,` as separator? We had it there for "backward compatibility" for 2 years at least.

Reply:
I don't see any problem with supporting both (plus space). Why `:` anyway? `,` makes more sense to me

Reply:
`:` is standard shell separator, right? Used in `PATH` for example. I am ok to leave both of them, if it makes sense to you.

Reply:
`:` is used in path search lists for some historical reason. for a general list of things `,` makes more sense :shrug: